### ISSUE TITLE
[Feature Pitch] Quarry rebalance for less drastic game impact - faster at mid (specifics in comments)

### ISSUE THREAD

**Issue by MrPotatobird** [transhumandesign/kag-base#147]
![quarry2](https://user-images.githubusercontent.com/39929457/41806395-4bb9ae56-768b-11e8-9952-545da93a58cf.gif)

TLDR / "what does it do," this quarry version dispenses 100 stone (for 50 coins) once every 60-180 seconds based on how close it is to the center of the map. Even the maximum rate is much slower than the current quarries.

When built, the quarry will immediately start working, and then 1-3 minutes later it will stop and wait for someone to pay 50 coins. Once someone pays, it drops 100 stone and then starts working again.

Safe quarries built by weenies at the edge of the map will work slowly relative to manly quarries built on the front lines by cool people. This reduces the quarry's wicked and unholy minecraft/stalemate powers, but it's still kinda nice to have around even at base.

Also, cuz it takes no fuel and is less potent, price change to 150 wood for frame + (50 (rather than 0) wood, 100 (rather than 150) stone, 50 gold)

And it has a lantern that lights up when the stone is available so that's pretty cute :)

---

"y tho"

Everyone hates quarries right now, but I think it would be a bit sad to just poof them right back out of the game. If we just nerf the numbers of the current quarry, it may go from "busted" to "actively harmful wood-sucker nobody wants to use." So my goal here is a quarry that is weak / definitely not OP, but is something that might actually still be nice to have.

If you build them at your base, each one of these quarries will take roughly *3 minutes* to prepare 100 stone, that then requires someone to buy it for 50 coins before it makes any more. One person can get more than twice as much stone for free by resupplying over that amount of time.

Even if you build them right at mid, where they are 3 times faster than they are at the edge of the map, each quarry is still only a little faster than one person's resupply.

Can anyone call this OP? They probably can and will, but it seems pretty tame to me.

Would you still use it? I think I would. I'd totally buy occasional little bursts of 100 stone for 50 coins. It would be a nice option.

I was purposely being cautious with the numbers. I think you could probably halve the time and price and it would still be okay, but better safe than sorry at this point.

---

Problems/annoyances with current quarry I think this solves:

- Quarry leads to big stone minecraft fortresses and prolongs games
	- If quarries built on the front lines are more efficient, this further rewards teams that have control of mid, thus mitigating some of the defensive power of quarries
	- If they produce stone VERY slowly (slower than other available methods) they shouldn't have a huge impact on fortress size
- Builders who use the vast amounts of stone from quarry get rich by playing it safe
	- Each dollop of 100 stone costs 50 coins, only up to 40 of which you can get back by building with the stone, so you're losing some coins instead of getting rich. This is one reason why it's better to cost coins than wood
- Players have to waste time by constantly tree farming and feeding the quarries to make sure they're getting the most stone they can from them
	- You can't really waste your time with these. You just plunk it down and then leave it to go play the game, fight some guys, whatever, and come back later for a little stone if you want.
- Because it's tied to wood, it makes wood and tree farms much more valuable, which is bad because the wood economy is basically "as much wood as you want" and is balanced because unlimited wood is not a problem as any amount of wooden structure is weak without stone to reinforce it.
	- Separate stone generation + wood generation. Wood is still just as lovably crappy as it always was. Stop by a tree farm every few minutes to get like a thousand wood for a few seconds of your time, stop by a quarry every few minutes to get a bit of stone for a big chunk of change. This is much more in the spirit of "the stone version of a tree farm:" it's like a tree farm, but it's much stingier because stone is more precious.

Additional benefits:
- Encourages risky gold placement and thus gives "losing" team opportunities to steal gold, like with tunnels. Pillaging the gold from structures is fun and not something that happens too often at the moment.
- Knights/archers might also want to quickly pay to drop the stone for any impoverished (or just not nearby) builder teammates to keep the stone coming. Ah, the beauty of teamwork and friendship.
- Because the price is at the end of its production cycle, the one who "pays" for it to make stone is actually likely to be the one to reap the benefits (instant gratification) so it feels better to use.

Potential issues:
- It's still extra stone generated that would not otherwise be in the game.
	- This is true, but when it's stone at the level of "less than if one deadweight momo on your team actually bothered to resupply" it can't be too bad can it? Resupplying is already an infinite but slow source of stone generation anyway.
- The "more efficient at mid" thing is unintuitive and overcomplicated.
	- I've heard this from a few people but I really don't see why it's so out there. It's literally right there in the tooltip, you can even SEE the better quarries moving faster because the animation is speedier, like, people will figure it out. There are things in this game that are way less intuitive. And even if you don't know this, it's not a huge deal, you can just build them anywhere and they'll still function.

**Comment by MrPotatobird** [comment]
If you don't like the "faster at mid" part, we can just take that out, make the efficiency range 180-180 seconds or something, but I feel that while that part would be nice, the other changes are also things that need to happen.

Faster near the center of the map: Yes 👍  or No 👎 ?

**Comment by 1bardesign** [comment]
Seems cool but a reasonably big change to quarries haha. We'll try a simpler nerf in the short term but it's nice to see detailed, well presented pitches :+1:

**Comment by MrPotatobird** [comment]
Well, quarries themselves were a big change, so the way I see it compared to pre-quarry game this should make quarries a smaller change.

If we're gonna just nerf the numbers, I'd say go for a hard nerf to max fuel limit + time taken, and maybe add a cap to the number of buildable quarries if possible. I wouldn't nerf conversion ratio or gold cost, it's just going to make quarries more annoying, wood hungry, and high maintenance, when the main problem is mind-boggling stone output.
